app/org/sagebionetworks/bridge/services/StudyService.java

```diff
@@ -268,6 +287,7 @@ public Study createStudy(Study study) {
         study.setActive(true);
         study.setStrictUploadValidationEnabled(true);
         study.setEmailVerificationEnabled(true);
+        study.setEmailSignInEnabled(false);
```

Comment:
Is it worth adding a unit test to verify isEmailSignInEnabled cannot be set to true when created?